package.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -x
 
+BCPROV_JAR=bcprov-jdk15-139b01.jar
+BCPROV_URL=https://www.bouncycastle.org/betas/
```

Comment:
Maybe we can use the maven repo: https://repo1.maven.org/maven2/org/bouncycastle/bcprov-jdk15on/1.56/bcprov-jdk15on-1.56.jar